cpp/common/src/codingstandards/cpp/Class.qll

```diff
@@ -192,7 +192,10 @@ class TrivialMemberFunction extends IntrospectedMemberFunction {
  * class.
  */
 class TemplateOrTemplateClassMemberFunction extends MemberFunction {
-  TemplateOrTemplateClassMemberFunction() { isFromUninstantiatedTemplate(_) }
+  TemplateOrTemplateClassMemberFunction() {
+    isFromUninstantiatedTemplate(_) or
+    isFromTemplateInstantiation(_)
```

Comment:
issue #382 says we might want
```suggestion
(
      isFromUninstantiatedTemplate(_) or
      isFromTemplateInstantiation(_)
    ) and
    not this.isCompilerGenerated()
```

Reply:
For the current usage and query this isn't necessary - as we are using this class to _exclude_ cases, not include. As a belt-and-braces measure, we can add it in.